README.md

```diff
@@ -1,8 +1,49 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# John Cao . Jarvis Consulting
 
+I am a recent graduate from York University's Lassonde School of Engineering with a Bachelor's degree in Computer Science. I also hold a Bachelor's degree in Chemical Engineering from the University of Toronto. During my time at York, I have worked as part of the school's IT department where I acted as the main source of support for several business applications. I also spent 16 months working as an intern at Hydro One Telecom where I was able to speed up numerous tasks and processes through building scripts in Python and Bash. Through these various experiences, I was able to demonstrate my problem solving and communication skills. My interest in pursuing a career in software started when I took an Introduction to Programming course during my first undergrad. I was drawn to how programming allowed me to create and build things in a very accessible manner. Just a couple of lines of code and I was able to see the results of my work. Currently I am an Applications and Data Engineer at Jarvis where I have had the opportunity to improve my level of proficiency in various tools and technologies such as Java, Linux, and Docker as well as learning about good software development practices. I am looking forward to bring my skills and knowledge and contribute to your team.
```

Comment:
please run a grammarly check for the entire profile
![image](https://user-images.githubusercontent.com/58951265/129414795-7e5cd356-2716-4bf6-b2e2-f5399f627fcf.png)